service/history/transfer_queue_factory.go

```diff
@@ -155,8 +158,11 @@ func (f *transferQueueFactory) CreateQueue(
 		f.MatchingClient,
 		f.VisibilityManager,
 	)
+	if f.StandbyExecutorWrapper != nil {
+		standbyExecutor = f.StandbyExecutorWrapper.Wrap(standbyExecutor)
+	}
```

Comment:
If someone implements a new queue factory, they need to be sure to also depend on and use the active and/or standby executor wrappers. I think this presents a potential pitfall because it wouldn't be immediately obvious to the author of the executor wrapper why it isn't being applied to some executors.